### DISCUSSION TITLE
Does Edge/WebView2 have a deeper network-level trace?

### DISCUSSION THREAD

**Discussion by brianclink** [MicrosoftEdge/WebView2Feedback#3881] [Q&A]
I am having an issue I am trying to diagnose.  Edge is hanging on a particular URL, and in the Network tab of F12 it is showing 'Pending'.  However on the server side it appears it is sending the response, and the 'client' is closing the connection.  In the Wireshark trace, it appears that application data is being exchanged (it is encrypted), and the client is initiating the FIN.  I think it is related to HTTP/2 and SSL and (manually-entered) IPv6 as the issue does not reproduce on HTTP, and does not reproduce through a proxy (Fiddler) that downgrades to HTTP/1.1, and does not reproduce using an IPv4 address.

I am asking here because the same issue reproduce in WebView2, used in the git sample 'WebView2Browser'.  I imagine that your team may know of some internal diagnostics that I could use to try to determine what Edge is really doing under the covers, as the Network tab of the browser simply says 'Pending'.

![image](https://user-images.githubusercontent.com/4602264/121574516-55958e80-c9f4-11eb-8108-d7c918e9d604.png)

**Comment by champnic** [comment]
Hey @brianclink - Please take a look at the instructions here: https://dev.chromium.org/for-testers/providing-network-details

Let me know if you have other questions after checking those out. Thanks!

**Comment by ericlaw1979** [comment]
I wrote this up a bit more fully (with a video) here: https://textslashplain.com/2020/01/17/capture-network-logs-from-edge-and-chrome/ and sketched out some analysis tips in https://textslashplain.com/2020/04/08/analyzing-network-traffic-logs-netlog-json/
